rfcs/0050-merge-bot-for-maintainers.md

```diff
@@ -74,10 +108,9 @@ a Nixpkgs-core and use e.g. Flakes for leaf packages.
 [unresolved]: #unresolved-questions
 
 Should the possibility of merging PR's be part of @grahamcofborg or should it be a
-separate bot.
+separate bot. For now it is assumed it would be part of @grahamcofborg.
```

Comment:
Not sure whether we decided to actually do this as part of ofborg or in a separate bot.